lib/definitions/retry.js

```diff
@@ -2,9 +2,9 @@
  * Default exponential backoff configuration for retries.
  */
 const RETRY_CONF = {
-  retries: 3,
-  retryAfterBaseValue: 1000,
```

Comment:
these are [default values already set in Octokit](https://github.com/octokit/plugin-retry.js/blob/7dc13d95447c1828eea1f173a635289872d3b1bd/src/index.ts#L12-L15).